doc-content/jbpm-docs/src/main/asciidoc/BPMN2/create-service-task-proc.adoc

```diff
@@ -1,38 +1,53 @@
 [id='create-service-task-proc']
 
 = Creating service tasks
-A service task is a task that is part of the process that is executed outside of the process, but is not a human task. Examples of service tasks include sending an email and logging a message when these tasks are performed by systems. You can define the parameters (input) and results (output) that are associated with a service task. A Service Task should have one incoming connection and one outgoing connection.
+A service task is a task that executes an action based on a web service call or in a Java class method. Examples of service tasks include sending an email and logging a message when these tasks are performed by systems. You can define the parameters (input) and results (output) that are associated with a service task. A Service Task should have one incoming connection and one outgoing connection.
```

Comment:
Missing part about Web Services and importing Java types from it. All communication between Business Central and Web Services is done using Java types from Web Services so we need to import types to the Business Central and add uploaded artefact to the Project. See important note number two: https://github.com/bpmn-tutorials/Imports

Reply:
@hasys we have covered the .jar upload and adding dependencies in the procedure.

Reply:
I can see it in procedure for Java Service Tasks but I can't see it for Web Service Tasks.
* For Java Service Tasks you upload Java **Classes** which will be executed.
* For Web Service Task you upload Java **Types** (Technically it is also classes but used as Data Objects - objects with data and without logic) which will be used during communication Business Central (kie server) <=> Web Service,

Reply:
@hasys thanks for explaining. I have added the common steps of uploading .jar file and adding dependency following by separate set of steps for web service and Java service task. Also, I have added a note related to .jar file that describes the difference for web service and java service task. 
Please take a look and let me know if this is correct.